### ISSUE TITLE
Add null alt to placeholder poster images

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/sul-embed#1738]
The `<img>` elements for the `locked-media-poster.svg` and the `waveform-audio-poster.svg` should have a null alt: `alt=""` or simply `alt`. (Otherwise, the screen reader will read out the entire filename.) The null alt is acceptable here, since these images are merely adding visual decoration.
